lib/pages/timetable/service/uni_event_provider.dart

```diff
@@ -332,6 +332,19 @@ class UniEventProvider extends EventProvider<UniEventInstance>
         .partDayEvents);
   }
 
+  Future<Iterable<UniEventInstance>> getUpcomingEvents(LocalDate date) async {
```

Comment:
Can you test out how this behaves during a holiday/exam session please? You can change the info in the dev db (careful not to break prod!) in `calendars > 2020`.

Reply:
Shows "None yet" during holiday/exam with no events inside.
In the DB I inserted a 'Mid term' examSession, and it shows the events that are also happening in this time frame

Reply:
Ideally it should show you that you're on holiday. It can show like another event (as they do when you open them in the calendar), or you can define a custom string that shows instead of "None yet" (like "You're on holiday, enjoy!" or idk). SImilarly, "None yet" is kind of misleading when you actually have events but they're further than say, 1 week away. Maybe show something like "You're all caught up!"?

Let's ask Ioana about this today.